test/fixtures/specs/element.spec.js

```diff
@@ -12,6 +12,11 @@ describe('element as first class citizen', () => {
         expect(header.getLocation('y')).to.be.equal(20)
     })
 
+    it('should be able to use hasFocus on element', () => {
+        let login = browser.element('[name=login]')
+        expect(login.hasFocus()).to.be.true
```

Comment:
Interesting...  I got that from using `execute` to get the name of `document.activeElement`!